docs/pages/make/sharepoint-agents/01-first-agent.md

```diff
@@ -2,6 +2,237 @@
 
 ---8<--- "msa-labs-prelude.md"
 
-Under construction ...
+In this lab, you'll create a SharePoint agent to work with documents stored in SharePoint Online. The agent that you are going to create helps employees of an hypothetical company to get information, policies, and rules from the HR department. The knowledge base of the agent will be a set of documents stored in a SharePoint Online document library.
+
+## Exercise 1: Upload sample documents
+
+In this step you will upload sample documents which will be used by your SharePoint agent to respond to user prompts. These include some fictitious Word, PowerPoint, and PDF files.
+
+### Step 1: Create a SharePoint site
+
+Within the [Microsoft 365 app](https://www.office.com/){target=_blank}, or elsewhere in Microsoft 365, click the "waffle" menu 1️⃣ and select **SharePoint** 2️⃣.
+
+![The UI of Microsoft 365 with the waffle menu expanded and the SharePoint workload highlighted.](../../../assets/images/extend-m365-copilot-05/upload-docs-01.png)
+
+Then select **Create Site** 1️⃣ and choose a **Team site** 2️⃣.
+
+![The UI to create a new SharePoint Online site, with 'Team Site' template suggested.](../../../assets/images/extend-m365-copilot-05/upload-docs-02.png)
+
+Select the **Standard team** site template; you will be shown a preview of the site. Select **Use Template** to continue.
+
+![The UI to select the 'Standard' site template for the target site.](../../../assets/images/extend-m365-copilot-05/upload-docs-03.png)
+
+Give your site a name such as "Copilot Dev Camp - HR" 1️⃣ and select **Next** 2️⃣.
+
+![The UI to provide name, description, and other details for the target site to create.](../../../assets/images/make/sharepoint-agents-01/create-site-01.png)
+
+Then select your privacy settings and language, and select **Create Site**.
+
+![The UI to select the privacy settings and the language for the target site.](../../../assets/images/make/sharepoint-agents-01/create-site-02.png)
+
+Skip adding any new members to the site and select **Finish** when the site provisioning is done. After a few moments, you will be presented with a new SharePoint site. 
+
+<cc-end-step lab="msa1" exercise="1" step="1" />
+
+### Step 2: Upload the sample documents
+
+Download this zip file consisting of few files (Word, PowerPoint, PDF) by selecting this [link](https://download-directory.github.io/?url=https://github.com/microsoft/copilot-camp/tree/main/src/make/copilot-studio/HR-documents&filename=hr-documents){target=_blank} and unzip the file somewhere on your local file system.
```

Comment:
Zip link didn't work for me in the first try, it worked in the second time though. (Optional) Maybe we can just put the zip file in the folder and redirect to raw url?